tools/multipleJobsToTschenggins.sh

```diff
@@ -29,7 +29,13 @@ function update {
 	curl --header "Content-Type: application/json" -s \
 		--request POST \
 		--data '{"cmd":"update","debug":0,"states":[{"name":"'$3'","result":"'$1'","server":"'$TARGETSERVER'","state":"'$2'"}]}' \
-		http://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/ng/ > /dev/null
+		https://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/  \
+        --cacert tsueri.oo.crt --insecure \
```

Comment:
No leaks please! Remove any reference to our existing setup. Perhaps make it a command line option.